flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -2332,7 +2332,8 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
 
     public static final BuiltInFunctionDefinition TO_TIMESTAMP_LTZ =
             BuiltInFunctionDefinition.newBuilder()
-                    .name("TO_TIMESTAMP_LTZ")
+                    .name("toTimestampLtz")
+                    .sqlName("TO_TIMESTAMP_LTZ")
```

Comment:
reverted the change of function name. Added necessary logic register the function in this commit.